example-server/app/public/form.js

```diff
@@ -558,8 +571,80 @@ app.controller('AutoSearchController', [
     scope.eventString = JSON.stringify(scope.event, null, 2);
     scope.error       = '';
 
+    scope.overrideApiKey = function () {
+      scope.allowOverrideApiKey = !scope.allowOverrideApiKey;
+    };
+
+    scope.overrideSearch = function () {
+      scope.allowSearchOverride = !scope.allowSearchOverride;
+      if (!scope.allowSearchOverride) {
+        scope.searchPath = '';
+      }
+    };
+
+    scope.overrideBeacon = function () {
+      scope.allowBeaconOverride = !scope.allowBeaconOverride;
+      if (!scope.allowBeaconOverride) {
+        scope.beaconPath   = '';
+        scope.manualBeacon = false;
+      }
+    };
+
+    scope.beaconPathChange = function () {
+      console.log('changed');
+      scope.manualBeacon = scope.beaconPath.length !== 0;
+    };
+
     scope.send = function () {
-      sendEvent(scope, sentTimeout, tracker, 'sendAutoSearchEvent');
+      try {
+        scope.jsonError        = false;
+        const searchBodyObject = JSON.parse(scope.searchBody);
+
+        let searchUrl;
+        if (scope.searchPath.length > 0) {
+          searchUrl = scope.searchPath;
+        } else {
+          searchUrl = "https://" + scope.customerId + "-cors.groupbycloud.com/api/v1/search";
+        }
```

Comment:
Could do `const search =  scope.searchPath.length > 0 ? scope.searchPath : ("https://" + scope.customerId + "-cors.groupbycloud.com/api/v1/search")` so that search is a const.